packages/core/src/external/carequality/ihe-gateway-v2/saml/saml-client.ts

```diff
@@ -36,7 +41,7 @@ export type SamlClientResponse = {
   success: boolean;
 };
 
-export async function getTrustedKeyStore(): Promise<string> {
+export async function loadTrustedKeyStore(): Promise<string> {
```

Comment:
nit: this one is not used anywhere else, so we can get rid of the export, right? It minimizes the exposed surface of this module/file and makes it easier to maintainers to know which functions are used internally and can more easily be changed.